mlir/utils/jenkins/Jenkinsfile

```diff
@@ -833,7 +848,7 @@ pipeline {
                             // Clean up build settings to disable static library and allow
                             // ROCm testing
                             sh 'rm build/CMakeCache.txt'
-                            buildProject('check-rocmlir-build-only ci-performance-scripts rocblas-benchmark-driver',
+                            buildProject("check-rocmlir-build-only ci-performance-scripts ${getGEMMBenchmarkDrivers()}",
```

Comment:
Out of curiosity, does copying the script requires that CK needs to be built ? (assuming all this does is copy the script apart from building ? )